src/internal/easee/model.go

```diff
@@ -0,0 +1,134 @@
+	LatestFirmware                               int         `json:"latestFirmware"`
+	Voltage                                      float64     `json:"voltage"`
+	ChargerRAT                                   int         `json:"chargerRAT"`
+	LockCablePermanently                         bool        `json:"lockCablePermanently"`
+	InCurrentT2                                  float64     `json:"inCurrentT2"`
+	InCurrentT3                                  float64     `json:"inCurrentT3"`
+	InCurrentT4                                  float64     `json:"inCurrentT4"`
+	InCurrentT5                                  float64     `json:"inCurrentT5"`
+	OutputCurrent                                float64     `json:"outputCurrent"`
+	IsOnline                                     bool        `json:"isOnline"`
+	InVoltageT1T2                                float64     `json:"inVoltageT1T2"`
+	InVoltageT1T3                                float64     `json:"inVoltageT1T3"`
+	InVoltageT1T4                                float64     `json:"inVoltageT1T4"`
+	InVoltageT1T5                                float64     `json:"inVoltageT1T5"`
+	InVoltageT2T3                                float64     `json:"inVoltageT2T3"`
+	InVoltageT2T4                                float64     `json:"inVoltageT2T4"`
+	InVoltageT2T5                                float64     `json:"inVoltageT2T5"`
+	InVoltageT3T4                                float64     `json:"inVoltageT3T4"`
+	InVoltageT3T5                                float64     `json:"inVoltageT3T5"`
+	InVoltageT4T5                                float64     `json:"inVoltageT4T5"`
+	LedMode                                      int         `json:"ledMode"`
+	CableRating                                  float64     `json:"cableRating"`
+	DynamicChargerCurrent                        float64     `json:"dynamicChargerCurrent"`
+	CircuitTotalAllocatedPhaseConductorCurrentL1 float64     `json:"circuitTotalAllocatedPhaseConductorCurrentL1"`
+	CircuitTotalAllocatedPhaseConductorCurrentL2 float64     `json:"circuitTotalAllocatedPhaseConductorCurrentL2"`
+	CircuitTotalAllocatedPhaseConductorCurrentL3 float64     `json:"circuitTotalAllocatedPhaseConductorCurrentL3"`
+	CircuitTotalPhaseConductorCurrentL1          float64     `json:"circuitTotalPhaseConductorCurrentL1"`
+	CircuitTotalPhaseConductorCurrentL2          float64     `json:"circuitTotalPhaseConductorCurrentL2"`
+	CircuitTotalPhaseConductorCurrentL3          float64     `json:"circuitTotalPhaseConductorCurrentL3"`
+	ReasonForNoCurrent                           int         `json:"reasonForNoCurrent"`
+	WiFiAPEnabled                                bool        `json:"wiFiAPEnabled"`
+	LifetimeEnergy                               float64     `json:"lifetimeEnergy"`
+}
+
+type ChargerMode int
+
+func (m ChargerMode) String() string {
+	switch m {
+	case 0:
+		return ChargerModeUnavailable
+	case 1:
+		return ChargerModeDisconnected
+	case 2:
+		return ChargerModeReadyToCharge
```

Comment:
This mode is I think not shown in the supported modes in the inclusion report. Also, you might be using there consts directly instead of just explicit strings.

Reply:
Good catch. Fixed.